server/cluster/cluster.go

```diff
@@ -496,6 +496,13 @@ func (c *RaftCluster) GetOperatorController() *schedule.OperatorController {
 	return c.coordinator.opController
 }
 
+// SetPrepared set the prepare check to prepared. Only for test purpose.
+func (c *RaftCluster) SetPrepared() {
+	c.coordinator.prepareChecker.Lock()
+	defer c.coordinator.prepareChecker.Unlock()
+	c.coordinator.prepareChecker.prepared = true
+}
```

Comment:
This function uses prepareCheck's lock to lock the update operator for prepareChecker's item, could you please make a new function for SetPrepared for `prepareChecker` directly?  Using its lock in `prepareChecker` can help us to protect the usage of lock and avoid deadlock.

Reply:
It's only used in tests.